Improved home screen, changed onboarding text, NCI Trial API recursion

alexaaalami opened the pull request:
# *Improved home screen, changed onboarding text, NCI Trial API recursion*

### Code of Conduct & Contributing Guidelines 

By submitting creating this pull request, you agree to follow our [Code of Conduct](https://github.com/StanfordBDHG/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordBDHG/.github/blob/main/CONTRIBUTING.md):
- [x] I agree to follow the [Code of Conduct](https://github.com/StanfordBDHG/.github/blob/main/CODE_OF_CONDUCT.md) and [Contributing Guidelines](https://github.com/StanfordBDHG/.github/blob/main/CONTRIBUTING.md).

alexaaalami committed:
Improved home screen, changed onboarding text, NCI Trial API recursion

PSchmiedmayer reviewed (approved):
Looks great; thank you @alexaaalami 🚀

alexaaalami committed:
Text edots

Minor text edits made to the "Welcome" page, as well as removal of the "Learn more..." link.

philippzagar commented:
Closing this stale PR for irrelevance
